app/controllers/admin/orders_controller.rb

```diff
@@ -1,9 +1,24 @@
 class Admin::OrdersController < AdminController
   def index
-    @orders = Order.all
+    @statuses = Status.all
+    @orders = params[:status_id] ? Status.find(params[:status_id])
+      .orders : Order.all
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.